master/buildbot/www/avatar.py

```diff
@@ -27,17 +27,27 @@
 class AvatarBase(config.ConfiguredMixin):
     name = "noavatar"
 
-    def getUserAvatar(self, email, size, defaultAvatarUrl):
+    def getUserAvatar(self, email, username, size, defaultAvatarUrl):
         raise NotImplementedError()
 
 
+class AvatarGitHub(AvatarBase):
+    name = "github"
+
+    AVATAR_URL = 'https://avatars.githubusercontent.com/'
+
+    def getUserAvatar(self, email, username, size, defaultAvatarUrl):
+        github_url = self.AVATAR_URL + username.decode("utf-8")
+        raise resource.Redirect(github_url)
```

Comment:
Thanks. Looks like this returns the full-size image. Is there a way to use the size parameter / should we be doing so?